akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKit.scala

```diff
@@ -185,6 +186,24 @@ final class ActorTestKit private[akka] (val name: String, val config: Config, se
    */
   def createTestProbe[M](name: String): TestProbe[M] = TestProbe(name)(system)
 
+  /**
+   * Verify serialization roundtrip.
+   * Throws exception from serializer if `obj` can't be serialized and deserialized.
+   *
+   * @param obj the object to verify
+   * @param assertEquality if `true` the deserialized  object is verified to be equal to `obj`,
+   *                       and if not an `AssertionError` is thrown
+   * @return the deserialized object
+   */
+  def verifySerialization[M](obj: M, assertEquality: Boolean): M = {
+    import akka.actor.typed.scaladsl.adapter._
+    val result =
+      SerializationExtension(internalSystem.toUntyped).verifySerialization(obj.asInstanceOf[AnyRef]).asInstanceOf[M]
+    if (assertEquality && result != obj)
+      throw new AssertionError(s"verifySerialization expected $obj, but was $result")
```

Comment:
```suggestion
      throw new AssertionError(s"Serialization verification expected $obj but was $result.")
```

The function name would be seen anyway.